docs/en/Running-Alluxio-Fault-Tolerant.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: global
-title: Alluxio Standalone with Fault Tolerance
-nickname: Alluxio Standalone with Fault Tolerance
+title: Alluxio on a Cluster with High Availability
+nickname: Alluxio on a Cluster with HA
```

Comment:
you mean for the navbar?

after my changes the navbar labels will be:
- Alluxio on Local Machine
- Alluxio on a Cluster
- Alluxio on a Cluster with HA

What would you prefer them to be?